src/main/java/frc/robot/subsystems/ShooterConfig.java

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+
+
+package frc.robot.subsystems;
+
+/** Add your docs here. */
+public class ShooterConfig {
+    //ShooterConfig [] DistanceArray;
+    double Distance;
+    double Angle;
+    double RPM;
+    
+
+    public ShooterConfig(double DDistance, double DAngle, double DRPM ){
+        this.Distance = DDistance;
+        this.Angle = DAngle;
+        this.RPM = DRPM;
+    }
+
+    public double getDistance(){
+        return this.Distance;
+    }
+    
+    public double getAngleDegrees(){
+        return this.Angle;
+    }
+    public double getVelocity(){
+        return this.RPM;
+    }
+    public double[] getVelocityAndAngle(){
+        double[] returnArray = new double[2];
+        returnArray[0] = getVelocity();
+        returnArray[1] = getAngleDegrees();
+        return returnArray;
+    }
+
+    public static double[] Interprolate(ShooterConfig obj1, ShooterConfig obj2,double distance){
```

Comment:
Interpolate is spelled incorrectly.

nit: It should also be lower case, as it's a method that is not a constructor